src/main/java/org/jvnet/hudson/test/ClosureExecuterAction.java

```diff
@@ -35,7 +35,7 @@
 import java.util.UUID;
 
 /**
- * Server-side logic that implements {@link HudsonTestCase#executeOnServer(java.util.concurrent.Callable)}.
+ * Server-side logic that implements {@link JenkinsTestCase#executeOnServer(java.util.concurrent.Callable)}.
```

Comment:
Since HudsonTestCase is deprecated but needed for binary compatibility, we should not rename the class.
```suggestion
 * Server-side logic that implements {@link HudsonTestCase#executeOnServer(java.util.concurrent.Callable)}.
```